plugins/dialog/plugin.js

```diff
@@ -862,8 +862,10 @@ CKEDITOR.DIALOG_STATE_BUSY = 2;
 			// Reset all inputs back to their default value.
 			this.reset();
 
-			// Select the first tab by default.
-			this.selectPage( this.definition.contents[ 0 ].id );
+			// Select provided tab or first one if there is nothing provided.
```

Comment:
It selects first tab if no tab is already selected. The comment is a little misleading.